spec/models/conference_spec.rb

```diff
@@ -778,6 +778,33 @@
     end
   end
 
+  describe '#scheduled_event_distribution' do
+    let!(:conference) { create(:conference) }
```

Comment:
it will be created when calling the variable, `let!` is not needed. Please read: https://www.relishapp.com/rspec/rspec-core/v/2-5/docs/helper-methods/let-and-let :wink: